src/python/grpcio_tests/tests_aio/unit/_test_server.py

```diff
@@ -30,11 +32,17 @@ class _TestServiceServicer(test_pb2_grpc.TestServiceServicer):
             sleep(test_constants.LONG_TIMEOUT)
 
 
-async def start_test_server():
+async def start_test_server(secure=False):
```

Comment:
Can we also add tests for certificate based secure channel? You can find credentials under `src/python/grpcio_tests/tests/unit/credentials`.